src/taxes/components/TaxChannelsMenu/TaxChannelsMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { Card } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+import ListItemLink from "@saleor/components/ListItemLink";
+import {
+  List,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+  makeStyles
+} from "@saleor/macaw-ui";
+import { taxesMessages } from "@saleor/taxes/messages";
+import { channelsListUrl } from "@saleor/taxes/urls";
+import clsx from "clsx";
```

Comment:
This should raise an error in eslint 🤔 I'll make a ticket for this.
```suggestion
import clsx from "classnames";
```